File: javascript/fizzbuzz.js

function fizzBuzz(input) {
    var returnValue = "";

    if (input % 3 == 0) {
        returnValue += "fizz";
    }
    if (input % 5 == 0) {
        returnValue += "buzz";
    }
    if (!returnValue) {
        return input;
    }
    return returnValue;
}

module.exports = fizzBuzz

File: java/src/main/java/FizzBuzz.java
public class FizzBuzz {
    public String fizzBuzz(Integer input) {
        String returnInput = (input % 3 == 0) ? "fizz" : "";
        returnInput += (input % 5 == 0) ? "buzz" : "";

        if (returnInput.isEmpty()) {
            return input.toString();
        }
        return returnInput;
    }
}

